essay_proposal.md

```diff
@@ -0,0 +1,12 @@
+# Essay Proposal
+I want to write an essay on Azure DevOps Services. 
+
+The essay will cover:
+- What is it?
+- Why use it?
+- How to use it?
+- Services and tools (Boards, Pipelines, Repos, Artifacts, Testing)
+- How it compares to other tools like Jenkins, JIRA, GitLab
```

Comment:
Azure is an IaS provider. It is not comparable to Jenkins, JIRA, GitLab.

Reply:
What I meant is to compare Azure DevOps Pipelines with Jenkins and Azure DevOps Boards with JIRA and Azure DevOps with Gitlab. Is this not possible?